src/components/form/cancel-button/cancel-button.js

```diff
@@ -0,0 +1,27 @@
+import I18n from "i18n-js";
+import React from 'react';
+import { tagComponent } from '../../../utils/helpers/tags';
+import Button from './../../button';
+
+let CancelButton = props =>
```

Comment:
Not sure if there is a preference over using or not using the implicit return style here as opposed to the style used in the functions defined below.